twine.json

```diff
@@ -50,12 +55,19 @@
             }
         }
     },
+    "input_manifest":{
+		"datasets": {
+            "aerofoil_shape_data":{
+                "purpose": "A dataset containing shape files in .dat or tab delimited open form, describing the aerofoil shape."
+            }
+        }
+    },
     "input_values_schema": {
         "title": "Input values schema",
         "type": "object",
         "properties": {
-            "airfoil_label": {
-                "title": "Airfoil label",
+            "airfoil_geometry_filename": {
+                "title": "Airfoil geometry filename",
```

Comment:
Can you add a description to this? e.g. what filenames are available? Where are the files coming from?